test/integration_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+yarn run build;
+
+# Interactive Tests
+for f in ./test/integration-tests/*.exp; do
+  expect "$f" > /dev/null 2>&1
```

Comment:
If you add `log_user 0` to your expect scripts, it will suppress output from the node command, which will free you up to remove this `> /dev/null` so that you can add diagnostic output to the script